nemo_skills/pipeline/utils.py

```diff
@@ -332,6 +350,13 @@ def check_remote_mount_directories(directories: list, cluster_config: dict, exit
         raise ValueError("Cluster config is not provided.")
     if isinstance(directories, str):
         directories = [directories]
+
+    # Get unmounted path of all directories
+    directories = [
+        get_unmounted_path(cluster_config, dir_path) if is_mounted_filepath(cluster_config, dir_path) else dir_path
```

Comment:
isn't the if part redundant? I think the get_unmounted_path already returns original if it's not mounted